set_tags.sh

```diff
@@ -0,0 +1,24 @@
+BRANCH="master"
+
+# Are we on the right branch?
+if [ "$TRAVIS_BRANCH" = "$BRANCH" ]; then
+  
+  # Is this not a Pull Request?
+  if [ "$TRAVIS_PULL_REQUEST" = false ]; then
+    
+    # Is this not a build which was triggered by setting a new tag?
+    if [ -z "$TRAVIS_TAG" ]; then
+      echo -e "Starting to tag commit.\n"
+
+      git config --global user.email "travis@travis-ci.org"
+      git config --global user.name "Travis"
+
+      # Add tag and push to master.
+      git tag -a v${TRAVIS_BUILD_NUMBER} -m "Travis build $TRAVIS_BUILD_NUMBER pushed a tag."
+      git push origin --tags
+      git fetch origin
```

Comment:
when is that triggered?

Reply:
1- for a given branch,
2- not a pull request,
3- not triggered by a tag!

Reply:
i know how to read but that doesnt help

Reply:
we could do whatever we want! but do we need to deploy apk foreach branch and commit?